src/main/java/botimusprime/tasks/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package botimusprime.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDateTime deadline;
+
+    public Deadline(String description, boolean isDone, LocalDateTime deadline) {
+        super(description, isDone);
+        this.deadline = deadline;
+    }
+
+    @Override
+    public boolean isEvent() {
+        return false;
+    }
+
+    @Override
+    public boolean isToDo() {
+        return true;
+    }
+
+    @Override
+    public boolean isDeadline() {
+        return false;
+    }
```

Comment:
The method isToDo returns true, while isDeadline returns false, which seems contradictory given the class is named Deadline. This might be a mistake.

Reply:
Oh my goodness, good spot! I missed this completely, thanks so much